src/components/Tiles/MovieTile.scss

```diff
@@ -0,0 +1,124 @@
+@import './variables';
+
+
+
+
+.movie-tile {
+    width: 335px;
+    height: 579px;
+    float: left;
+    margin: 10px;
+    position: relative;
+
+
+
+    .movie-details {
+        position: static;
+        background-color: $background-movie-hover;
+        opacity: 0%;
+        color: $font-primary-color;
+        width:100%;
+        height: 100%;
+        transition: opacity 1s;
+        -webkit-transition: opacity 1s;
+        -moz-transition: opacity 1s;
+        -ms-transition: opacity 1s;
+        -o-transition: opacity 1s;
+        padding: 20px;
+        box-sizing: border-box;
+
+    }
+    .img-hover {
+        width:100%;
+    }
+    .movie-details:hover {
+        opacity: 90%;
+    }
+
+    .img-tile {
+        position: absolute;
+        object-fit: contain;
+        width: 100%;
+        height: 100%;
+        
+    }
+
+    .title {
+        position: relative;
+        margin-bottom: auto;
+        text-align: center;
+        background-color: $btn-color;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        color: $font-primary-color;
+        width: 100%;
+        padding: 10px 5px;
+        box-sizing: border-box;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+
+}
+
+@media all and (max-width:425px) {
```

Comment:
We have media queries defined with mixins in project, use them

Reply:
Now code is using them